.github/workflows/issue-triage.yml

```diff
@@ -18,3 +18,23 @@ jobs:
         with:
           repo-token: '${{ secrets.GITHUB_TOKEN }}'
           add-labels: 'Status: Triage'
+      - name: Check for duplicate issues
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { data: issues } = await github.issues.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              state: 'open',
+              labels: 'Status: Triage'
+            });
+            const duplicate = issues.find(issue => issue.title === context.payload.issue.title);
```

Comment:
I love the idea of finding duplicates automatically, we can move this out into a separate PR, with a more powerful script, which would address some issues of this approach. 

- The chance of exact matches in title would be very low. 
- listForRepo would only return 30 items without pagination. 
- Searching for `labels: 'Status: Triage'` would only give a subset of issues, there might be open/closed issues for the same request.